### ISSUE TITLE
(feat) also output full parameters functions aliases (/2 /3 ...)

### ISSUE THREAD

**Issue by lissana** [hamler-lang/hamler#349]
currently the compiler generates this functions, (useful for composition on hamler):

'and'/1 =
    fun(_0) ->
        fun(_1) ->
            let <_2, _3> = <_0, _1>
            in case <_2, _3> of
                   <'true', 'true'> when 'true' ->
                       'true'
                   <_100, _101> when 'true' ->
                       'false'
               end

can have it generate the /2 alias too, something like (useful for calling hamler from other langs), hamler itself wouldn't use or call these, although it could as a compiler optimization

'and'/2 =
            let <_2, _3> = <_0, _1>
            in case <_2, _3> of
                   <'true', 'true'> when 'true' ->
                       'true'
                   <_100, _101> when 'true' ->
                       'false'
               end

**Comment by emqplus** [comment]
@lissana Yes. It's inconvenient to call the functions generated by Hamler in Erlang/Elixir now, for the functions in Hamler are curried when compiled to CoreErlang. 

@EMQ-YangM could we generate uncurried functions in CoreErlang?
